ATOI and ITOA/testing_itoa_atoi.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from atoi_itoa import atoi_itoa
+
+class Test(unittest.TestCase):
+
+    def test_atoi(self):
```

Comment:
Ok there are other bugs, but I want to give you an opportunity to come up with more test cases before I identify them :)